plugins/microsoft-teams/__tests__/microsoft-teams.test.ts

```diff
@@ -1,55 +0,0 @@
-import { dummyLog } from "@auto-it/core/src/utils/logger";
-import { sanitizeMarkdown } from "@auto-it/slack";
-import createHttpsProxyAgent from "https-proxy-agent";
-
-import MicrosoftTeamsPlugin from "../src";
-
-const fetchSpy = jest.fn();
-// @ts-ignore
-jest.mock("node-fetch", () => (...args) => {
-  fetchSpy(...args);
-});
-
-beforeEach(() => {
-  fetchSpy.mockClear();
-});
-
-const mockAuto = {
-  git: {},
-  logger: dummyLog(),
-} as any;
-
-describe("postToMicrosoftTeams", () => {
```

Comment:
It looks to me like those 2 tests are identical to the ones in `index.test.ts`:

- "should call Microsoft Office Teams api with minimal config"
- "should call Microsoft Office Teams api through http proxy"